### ISSUE TITLE
EROM data needing fixing

### ISSUE THREAD

**Issue by rburghol** [HARPgroup/r-dh-ecohydro#86]
Some points had bad matches with EROM (or old version of NHD or EROM had bad intersection), or EROM was screwy due to bad connectivity.  

Example:
- Fish Site: DGIF302138
- NHDplus COMID: 8525447 
- Mean Annual; 1.54 cfs
- Issue: from map, HUC look like it has considerable upstream drainage so should have higher Q, and fish site has NT = 22, so that suggests higher flow site as well.

![image](https://user-images.githubusercontent.com/4571170/43659319-a48a6fc0-9729-11e8-8617-3f91baf85b35.png)

![image](https://user-images.githubusercontent.com/4571170/43659440-0a732c8c-972a-11e8-8f7c-3e2e77af330e.png)

**Comment by jenniferRapp** [comment]
Rob,
what does bad connectivity mean?  I take a look at my data layer that I did
to join the points to the stream and  see what my joined EROM dataset would
have had for the flow there.  I don't see an NHDPlus stream anywhere in the
map view, so I wonder how far away that point had to be associated with the
nearest stream.  Are you planning to investigate all the HUC10 points that
may have these anomalies?




*Jennifer Lynn Rapp, Biogeographer and GIS Specialist*
jrapp@usgs.gov
U.S. Geological Survey
Virginia and West Virginia Water Science Center
1730 East Parham Road
Richmond, VA 23228
w. 804-261-2635

**Comment by rburghol** [comment]
"Bad connectivity" refers to the linkage upstream for the NHDplus watershed segment.  Based on the EROM data that we have for this location, the stream shows an area of about 1.0 square mile, and then the EROM flow which is 1.54.  But, looking at the topographic map here, this segment of NHDplus receives flow from upstream segments, but it is not represented as such in our version of EROM.

What draiange area and flow does your version of EROM show for this watershed/comid? (maybe we have an error in our version?).

As for QA'ing, this is me implementing the process that will be used as we move forward at DEQ.  i.e., you identify a location, look at available data, QA if need be, and do analysis.  We have done this all along at the HUC8 and HUC6 level (but not as much as we would have liked of course), but this is the first time (to my knowledge) that we are doing at the HUC10.

**Comment by jenniferRapp** [comment]
All these characterizations match with the NHD spatial joins I conducted.
The DA was 1.687 square miles and the MAF was 1.54.  It looks like it
snapped to the correct stream reach, given the location on the landscape
and the proximity to the streams.  The only thing I can guess is that for
some reason the data don't fit with the nhd stream characterizations is if
the LAt long were slightly off.  There are numerous VDGIF sample sites on
the mainstem just north of the sample in question.  If this site had a lat
of 37.51 it would be plotting on Craig Creek with 250 square miles DA, not
the small 2nd order trib.  Do you all have attribute data from VDGIF that
states the stream the data were collected on? It may be in EDAS.

Jen








*Jennifer Lynn Rapp, Biogeographer and GIS Specialist*
jrapp@usgs.gov
U.S. Geological Survey
Virginia and West Virginia Water Science Center
1730 East Parham Road
Richmond, VA 23228
w. 804-261-2635

**Comment by rburghol** [comment]
Good idea on EDAS - I will check that.  What I am actually thinking is that the NHDplus looks erroneous here, as I think it's missing an upstream trib -- but an error in the DGIF lat/lon is certainly a good possibility.  I will report back if I learn anything.

**Comment by rburghol** [comment]
Alas. Stream name not indicated in EDAS for this record.

**Comment by jenniferRapp** [comment]
bummer.  We could ask VDGIF.

Jen


*Jennifer Lynn Rapp, Biogeographer and GIS Specialist*
jrapp@usgs.gov
U.S. Geological Survey
Virginia and West Virginia Water Science Center
1730 East Parham Road
Richmond, VA 23228
w. 804-261-2635
